atomicapp/nulecule/main.py

```diff
@@ -40,25 +39,32 @@ def __init__(self, app_spec, destination=None):
         self.app_path = None  # The path where the app resides or will reside
         self.image = None     # The container image to pull the app from
 
-        # Doesn't make sense to provide a local path and a destination
-        if os.path.exists(app_spec) and destination:
+        # Adjust app_spec and destination paths if absolute.
+        if os.path.isabs(app_spec):
+            app_spec = os.path.join(Utils.getRoot(), app_spec[1:])
+        if destination and os.path.isabs(destination):
+                destination = os.path.join(Utils.getRoot(), destination[1:])
+
+        # Determine if the user passed us an image or a path to an app
+        if not os.path.exists(app_spec):
+            self.image = app_spec
+        else:
+            self.app_path = app_spec
+
+        # Doesn't make sense to provide an app path and destination
```

Comment:
So with the changes that I am making in this PR (see commit message for more context) it doesn't make sense to provide two paths. If you already have a path local then you would just use that as the location of the app.. if a user wants to make a copy of a local application they can do that with /bin/cp.